### DISCUSSION TITLE
求大灾变64位

### DISCUSSION THREAD

**Discussion by luqiyihui** [Gamuxorg/bbs#388] [投递和请求游戏]
版主，能帮忙打包各大灾变CDDA，64位版本的吗？
我的系统是arch
我在https://github.com/CleverRaven/Cataclysm-DDA/tree/master这个网站 git clone了文件，然后进入下载的目录
make
sudo make install

我在窗口桌面里用终端运行游戏，发现在小人周边约3-4格无显示（包括中文版和英文版），小人移动走后，原来区域显示正常，如果我纯文本命令行下运行游戏（开机不进入桌面），小人周边显示都正常（英文，和中文版都一样，只是中文显示成小方块）。
版主能帮忙打包一个给我下，实在能力有限解决不了问题才伸手的，或能指导下是不是我某个地方没有注意到？

先谢谢了！

**Comment by wangling12** [comment]
看了一下，这游戏太硬核了。。。。。
而且我没有Arch环境，不过打包了一个我当前系统看起来一切正常的
网盘太慢，正在传

**Comment by luqiyihui** [comment]
非常感谢！
坐等链接了：-）

**Comment by wangling12** [comment]
链接: https://pan.baidu.com/s/1c-MkEjHC8dZRF-uCNlyyEg 
提取码: 353u
先百度网盘凑合吧，有问题的话带上log

**Comment by luqiyihui** [comment]
[luqi@Archlinux ~]$ ./cataclysmdda-0.D-8574-Linux-Tiles.appimage 
./cataclysm-tiles: error while loading shared libraries: liblua5.3.so.0: cannot open shared object file: No such file or directory
Cleaning up
[luqi@Archlinux ~]$

**Comment by luqiyihui** [comment]
我已经安装过lua5.3了

**Comment by wangling12** [comment]
链接: https://pan.baidu.com/s/18PaNOxEtiY05pMNQQht8Ew 
提取码: ig55

**Comment by luqiyihui** [comment]
[luqi@Archlinux ~]$ ./cataclysmdda-0.D-8574-Linux-Tiles.appimage
Something went wrong trying to read the squashfs image.

Cannot mount AppImage, please check your FUSE setup.
You might still be able to extract the contents of this AppImage 
if you run it with the --appimage-extract option. 
See https://github.com/AppImage/AppImageKit/wiki/FUSE 
for more information
open dir error: No such file or directory
[luqi@Archlinux ~]$ ./cataclysmdda-0.D-8574-Linux-Tiles.appimage --appimage-extract
Something went wrong trying to read the squashfs image.
Failed to open squashfs image
[luqi@Archlinux ~]$

**Comment by wangling12** [comment]
https://pan.linuxgame.cn/index.php/s/GT5AawHWoggJeEX
![2019-05-27 22-31-12 的屏幕截图](https://user-images.githubusercontent.com/24770600/58446196-51121c00-8132-11e9-9592-d05bab3d3aef.png)

**Comment by wangling12** [comment]
对了，必须安装fuse，不然运行不了appimage

**Comment by luqiyihui** [comment]
谢谢，已经愉快的游戏了：-）
